apps/storybook/src/gp2-components/FilterPill.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import { FilterPill } from '@asap-hub/gp2-components';
+import { ComponentProps } from 'react';
+
+export default {
+  title: 'GP2 / Molecules / Filter Pill',
+  component: FilterPill,
+};
+
+const props: ComponentProps<typeof FilterPill> = {
+  value: {
+    id: 'region-Africa',
+    label: 'Africa',
+  },
+  onRemove: (value) => alert(`${value.label} erased!`),
```

Comment:
Not sure we should have this alert here.

The other ways I've seen this are with either a noop or a () => Promise.resolve()